### PR TITLE
Remove scaling properties from non-media elements

### LINKED ISSUE #12437: Code quality: Fix elements ending up with incorrect properties
## Bug Description
For some reason, text, shape and sticker elements end up with having `scale`, `focalX` and `focalY` properties.
Also, media elements end up with having `backgroundColor` property.

Figure out why this happens, add migration for removing those properties and update the TS types as well.

These were added to the `types` package in #12429 to reflect what we actually have, even though incorrect.

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

### PR DESCRIPTION
**Pull request by miina**
## Summary
Remove `scale`, `focalX`, `focalY` from elements that are not media. Apparently all the elements were added those properties.

## To-do

- [x] Check about `backgroundColor` ending up in media properties (haven't been able to repro yet)

## User-facing changes
N/A

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add non-media elements.
2. Open our development tools (`shift + option + cmd + J`)
3. Verify that the added elements don't include any of the removed properties

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #12437

### CODE REVIEW

File: packages/migration/src/types/element.ts [hunk 1]
```diff
@@ -86,6 +86,10 @@ export interface ElementV0 extends ElementBoxV0 {
   groupId?: string;
   border?: BorderV0;
   borderRadius?: BorderRadiusV0;
+
+  scale?: number;
```
Comment (miina) [inline packages/migration/src/types/element.ts:90]: Apparently all the elements had those added.

**Comment by swissspidy** [comment]
This was purely from typing the templates, so probably just leftovers there.

**Review by merapi — APPROVED** [review]
Don't worry about the failing Karma tests, I'm fixing them [here](https://github.com/GoogleForCreators/web-stories-wp/pull/12623).

**Comment by kkalarickal** [comment]
Confirmed that the `scale, focalX, focalY ` are removed from the story data with the `12605 branch` loaded. 

I created a test story with all types of elements embedded - please see the test story here: [Verify fix of #12605](https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/post.php?post=20660&action=edit#page=%252238e276c0-b11a-4c37-87f0-610b9ff593b6%2522)

- Image
- Video
- 3rd party image
- 3rd party video
- GIF
- Sticker
- Text Element
- Shape
- Sticker
- scaled image
- Shopping/product

Confirmed that `scale, focalX, focalY` are seen for 
- image
- video
- GIF
- Sticker
- scaled image

Confirmed that with `main branch` the other non-media elements ALSO had `scale, focalX, focalY`. But with the `12605 branch` these non-media elements do NOT have these properties.

ISSUE:

WITH FIX:

Also verified that scaling and moving the media elements DOES continues to work, and the `scale, focalX, focalY` data is updated in the story.

**Pull request merged**